### DISCUSSION TITLE
Error query

### DISCUSSION THREAD

**Discussion by shivers99** [Screenly/Anthias#1867] [Q&A] [answered]
Hi, I'm new to git and there's an argument that I probably shouldn't be here if I don'y know the answer this ;-) 
Anyway..
I'm getting an error at this section of the guide:

`git clone git@github.com:Screenly/Anthias.git`

Should I be using git@github.com? my username?
Trying both gets me:

Cloning into 'Anthias'...
shivers99@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

Do I need a public key in my git account?
Thanks

**Comment by vpetersson** [comment, marked as answer]
It needs to be git@ and you need to add your public key to your GitHub account.

Alternatively, if you just want to clone, just the http version.

**Reply by shivers99** [reply]
Thanks very much.
